### DISCUSSION TITLE
API endpoints

### DISCUSSION THREAD

**Discussion by wilek162** [DevSkillsHQ/discusssions#37] [Help] [answered]
The API's endpoints linked in the instructions does not seem to be found. /ping returns 404 not found.

**Comment by fenske** [comment, marked as answer]
Hi @WilliamEklundITHS. I'm Anton from Alva. To clarify, "The backend should implement the [Accounting API](https://infra.devskills.app/accounting/api/3.1.0)." means you need to build your own service that implements the referenced API spec. There is no deployed service behind it. The spec if for you to know what to build. Hope this clarifies it.

Let me know if you have any other questions.

I wish you the best of luck with the rest of your interview process!
